package.json

```diff
@@ -19,61 +19,57 @@
   },
   "scripts": {
     "build": "npm run clean && npm run transpile",
-    "clean": "del dist coverage",
+    "clean": "rm -rf dist && rm -rf coverage",
     "cover": "istanbul cover _mocha -- --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/index test",
     "docs": "esdoc -c esdoc.json",
     "lint": "npm run lint:src",
     "lint:src": "eslint src/**",
     "lint:test": "eslint test/**",
-    "preversion": "del node_modules && npm install && npm test",
+    "preversion": "rm -rf node_modules && npm install && npm test",
     "postversion": "git push && git push --tags",
     "test": "mocha --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/index test",
     "test:watch": "mocha -w --compilers js:babel-core/register -r babel-polyfill -s 100 --recursive test/index test",
     "transpile": "BABEL_ENV=production babel --no-comments --out-dir dist src"
   },
   "dependencies": {
-    "append-query": "2.0.1",
-    "core-js": "2.4.1",
+    "buffer": "5.0.6",
     "es6-error": "4.0.2",
     "es6-promise": "4.1.0",
+    "events": "1.1.1",
     "fast-memory-cache": "2.0.4",
-    "hellojs": "1.14.1",
```

Comment:
We will need to update the DevCenter to show how to connect Social Network accounts without using native APIs provided by the SDK.